scripts/config.ts.example

```diff
@@ -4,6 +4,8 @@ import { ethers } from 'ethers'
 // This need to be adjusted for Orbit chains
 export const maxDataSize = 117964
 
+export const eigenDACertVerifierAddress = '0x1234512345123451234512345123451234512345'
```

Comment:
which network?

Reply:
just a dummy value - this will be set in a separate `config.ts` when doing real deployments

Reply:
oh lol obviously I missed that its a dummy value.